src/main/scala/com/databricks/labs/overwatch/validation/DeploymentValidation.scala

```diff
@@ -378,6 +378,13 @@ object DeploymentValidation extends SparkSessionWrapper {
           config.eh_scope_key,
           config.eh_name,
           config.output_path)
+      case "gcp" =>
+        validateAuditLog(
+          config.workspace_id,
+          config.auditlogprefix_source_aws,
```

Comment:
This config needs to be changed globally to rename from `auditlogprefix_source_aws` to `auditlogprefix_source_path`. This will need to be handled in the DOCS and in the upgrade script. Please add two tickets for 0720 with the appropriate labels to make sure these get done and change the config name in this PR.

Reply:
Ticket created and assigned to 7.2.0
https://github.com/databrickslabs/overwatch/issues/802